src/views/operations/OperationsView.tsx

```diff
@@ -31,17 +31,20 @@ export const OperationsView = () => {
     }
   };
 
-  const loadingElement = (
-    <Text color={colors.gray[500]} textAlign="center" paddingY="20px">
-      Loading...
-    </Text>
-  );
-
   return (
     <Flex flexDirection="column" height="100%">
       <TopBar title="Operations" />
       {accountsFilter}
-      {isLoading && isFirstLoad && loadingElement}
+
+      <Center display={isLoading && isFirstLoad ? "flex" : "none"} height="100%">
+        <Image
+          width="150px"
+          height="75px"
+          marginBottom="136px"
```

Comment:
Why do we need marginBottom here? 🤔

Reply:
otherwise, it's not centerred against the whole screen, but rather only the part under the topbar